DotNet/Tenant/Services/FacilityConfigurationService.cs

```diff
@@ -316,12 +321,12 @@ private async Task MeasureDefinitionExists(String reportType)
 
                 string requestUrl = _serviceRegistry.Value.MeasureServiceUrl + $"/api/measure-definition/{reportType}";
 
-                if (_linkTokenServiceConfig.Value.SigningKey is null)
+                //TODO: add method to get key that includes looking at redis for future use case
+                if (!_linkBearerServiceOptions.Value.AllowAnonymous && _linkTokenServiceConfig.Value.SigningKey is null)
                     throw new Exception("Link Token Service Signing Key is missing.");
 
-
                 //get link token
-                if (_linkTokenServiceConfig.Value.EnableTokenGenerationEndpoint)
+                if (!_linkBearerServiceOptions.Value.AllowAnonymous && _linkTokenServiceConfig.Value.EnableTokenGenerationEndpoint)
```

Comment:
For now just leave it there. I'll make an architecture ticket to review how we want to do this because there might be a cleaner way to go about this. This is to mainly get the local compose working again.

Reply:
After discussing this it was decided to remove the `_linkTokenServiceConfig.Value.EnableTokenGenerationEndpoint` flag from the if statement.